src/CachePeer.cc

```diff
@@ -55,3 +57,32 @@ CachePeer::connectTimeout() const
     return Config.Timeout.peer_connect;
 }
 
+void
+CachePeer::peerConnectFailed(ACLFilledChecklist *checklist)
+{
+    debugs(15, DBG_IMPORTANT, "ERROR: TCP connection to " << host << "/" << http_port << " failed");
```

Comment:
We should not report errors that we are not responsible for, especially at level 1. Please adjust accordingly _after_ rebasing on top of master commit 92c4f62.